Remove "Early Access" from JetBrains IDEs section (#2953)
withastro/docs: src/content/docs/en/editor-setup.mdx

📚 See how to [set up TypeScript](/en/guides/typescript/) in your Astro project.

## JetBrains IDEs

> Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab and benefit from features like syntax highlighting, code completion, and formatting, with plans to add even more advanced features in the future. It is also available to all other [JetBrains IDEs with JavaScript support](https://www.jetbrains.com/products/#lang=js&type=ide).

Reviewer: ```suggestion
Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. This plugin includes features like syntax highlighting, code completion, and formatting, and plans to add even more advanced features in the future. It is also available to all other [JetBrains IDEs with JavaScript support](https://www.jetbrains.com/products/#lang=js&type=ide).
```
This was a loooong sentence, so I just broke it up a bit.

> JetBrains' upcoming [Fleet IDE](https://www.jetbrains.com/fleet/) also supports language servers and therefore, our currently available tooling will be able to run there without any problem.

Author: You're right, thanks!
